pages/en/blog/vulnerability/openssl-fixes-in-regular-releases-oct2023.md

```diff
@@ -0,0 +1,49 @@
+---
+date: 2023-10-25:00:15.000Z
+category: vulnerability
+title: OpenSSL Recent Security Patches
+slug: openssl-fixes-in-regular-releases-oct2023
+layout: blog-post.hbs
+author: Rafael Gonzaga
+---
+
+## Summary
+
+The vulnerabilities released in the OpenSSL Security Advisory of:
+
+- OpenSSL 3.0.10 - Tuesday 8th September 2023
+- OpenSSL 3.0.11 - Thursday 19th September 2023
+- OpenSSL 3.0.12 - Tuesday 24th October 2023
+
+do not affect any active Node.js release lines. Therefore, these patches will
+be released in regular releases.
+
+## Analysis
+
+Our assessment of the following security advisories:
+
+- [OpenSSL 3.0.10](https://mta.openssl.org/pipermail/openssl-announce/2023-July/000266.html)
+- [OpenSSL 3.0.11](https://mta.openssl.org/pipermail/openssl-announce/2023-September/000273.html)
+- [OpenSSL 3.0.12](https://mta.openssl.org/pipermail/openssl-announce/2023-October/000282.html)
+
+is:
+
+### POLY1305 MAC implementation corrupts XMM registers on Windows (CVE-2023-4807) - Low
+
+// TODO
```

Comment:
@tniessen @richardlau can you help me assess this report? I believe it does affect Node.js (on Windows), but I don't have much context on the functions we expose.